src/app/progress-bar/progress-bar.component.html

```diff
@@ -0,0 +1,2 @@
+<h2>We already sold {{progressValue}}% of out products</h2>
+<div class="progress-bar"><span class="progress" [style.width.%]="progressValue > 100 ? 100 : progressValue"></span></div>
```

Comment:
validation can be done in component - better way from Unit Test perspective

Reply:
Validation moved to service